Update Rundeck integration instructions (#2955)
DataDog/integrations-extras: rundeck/README.md

Other plugins, such as Advanced Run Job, can also be used, depending on your use case.

##### Agent configuration

1. Run the following command to install the Agent integration:
   ```shell
   datadog-agent integration install -t datadog-rundeck==1.1.0
   ```

>     > **Note**: For containerized environments, see the [Datadog documentation page][10]. 

Reviewer: ```suggestion
   **Note**: For containerized environments, see the [Use Community and Marketplace Integrations][10].
```

   
2. Configure `conf.d/rundeck.d/conf.yaml`. For more information, see [Agent Configuration Files][8].

>      - `url`: Rundeck URL

Reviewer: ```suggestion
    - `url`: Rundeck URL
```

>      - `access_token`: Rundeck API token

Reviewer: ```suggestion
    - `access_token`: Rundeck API token
```

> 

Reviewer 2: Maybe we can add something like this to Step 2 to clarify token setup?

```suggestion
    To generate an API token, follow the official Rundeck documentation:
    [https://docs.rundeck.com/docs/manual/10-user.html#generate-api-token](https://docs.rundeck.com/docs/manual/10-user.html#generate-api-token)

    Note: The API token must have permissions to access the following endpoints:

    `GET /metrics/metrics` [(ref)](https://docs.rundeck.com/docs/api/#metrics-data)
    `GET /system/info` [(ref)](https://docs.rundeck.com/docs/api/#system-info)
    `GET /project/[PROJECT]/executions` [(ref)](https://docs.rundeck.com/docs/api/#execution-query)
    `GET /project/[PROJECT]/executions/running` [(ref)](https://docs.rundeck.com/docs/api/#listing-running-executions)
    `GET /projects` [(ref)](https://docs.rundeck.com/docs/api/#listing-projects)
```

> 4. [Restart][9] the Agent.

Reviewer: ```suggestion
3. [Restart][9] the Agent.
```

> 5. Verify the installation:

Reviewer: ```suggestion
4. Verify the installation:
```
